sys/linux/fs_ioctl.txt

```diff
@@ -48,6 +51,14 @@ type fscrypt_policy_mode_t[CONTENTS, FILENAMES] {
 
 fs_policy_flags = FS_POLICY_FLAGS_PAD_4, FS_POLICY_FLAGS_PAD_8, FS_POLICY_FLAGS_PAD_16
 
+fsverity_digest_algorithm_flags = FS_VERITY_ALG_SHA256, FS_VERITY_ALG_SHA512, FS_VERITY_ALG_CRC32C
```

Comment:
These values are not in *.const files.
Did you re-run `make extract`? Or these consts are not yet in linux-next (assuming you using linux-next tree)?

Reply:
Fixed.  They are in linux-next, but note there is currently a push back for the uapi on the mailing list.